src/spyglass/lfp/v1/lfp.py

```diff
@@ -947,3 +949,35 @@ def fetch1_dataframe(self, *attrs, **kwargs):
             filtered_nwb["filtered_data"].data,
             index=pd.Index(filtered_nwb["filtered_data"].timestamps, name="time"),
         )
+
+    def compute_analytic_signal(self, electrode_list, **kwargs):
+        filtered_band = self.fetch_nwb()[0]["filtered_data"]
+        electrode_index = np.isin(filtered_band.electrodes.data[:], electrode_list)
```

Comment:
Could also use the spyglass nwb_helper_fn `get_electrode_indices` here to make sure the indexing is correct, but that has the downside of returning 99999999 or something if the electrode you pass to it isn't in the nwb_object.